Patches/Forsakens/Forsakens_CE_Patch_ThingDefs_Weapons.xml

```diff
@@ -0,0 +1,561 @@
+						<aiAimMode>AimedShot</aiAimMode>
+						<aimedBurstShotCount>3</aimedBurstShotCount>
+					</FireModes>
+
+					<weaponTags>
+						<li>ForsakenGun</li>
+					</weaponTags>
+				</li>			
+
+				<!-- == Shared patches for firearm melee tools == -->
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[
+						defName="ForsakenSniper" or
+						defName="ForsakenShotGun" or
+						defName="ForsakenAR"
+					]</xpath>
+					<value>
+						<tools>
+							<li Class="CombatExtended.ToolCE">
+								<label>stock</label>
+								<capacities>
+									<li>Blunt</li>
+								</capacities>
+								<power>8</power>
+								<cooldownTime>1.55</cooldownTime>
+								<chanceFactor>1.5</chanceFactor>
+								<armorPenetrationBlunt>2.755</armorPenetrationBlunt>
+								<linkedBodyPartsGroup>Stock</linkedBodyPartsGroup>
+							</li>
+							<li Class="CombatExtended.ToolCE">
+								<label>barrel</label>
+								<capacities>
+									<li>Blunt</li>
+								</capacities>
+								<power>5</power>
+								<cooldownTime>2.02</cooldownTime>
+								<armorPenetrationBlunt>1.630</armorPenetrationBlunt>
+								<linkedBodyPartsGroup>Barrel</linkedBodyPartsGroup>
+							</li>
+							<li Class="CombatExtended.ToolCE">
+								<label>muzzle</label>
+								<capacities>
+									<li>Poke</li>
+								</capacities>
+								<power>8</power>
+								<cooldownTime>1.55</cooldownTime>
+								<armorPenetrationBlunt>2.755</armorPenetrationBlunt>
+								<linkedBodyPartsGroup>Muzzle</linkedBodyPartsGroup>
+							</li>
+						</tools>
+					</value>
+				</li>
+				
+				<!-- ========== Forsaken Long Blade ========== -->
+
+				<!-- (CE implementation based on CE Melee Wraithblade) -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="ForsakenLongBlade"]/tools</xpath>
+					<value>
+						<tools>
+							<li Class="CombatExtended.ToolCE">
+								<label>handle</label>
+								<capacities>
+									<li>Blunt</li>
+								</capacities>
+								<power>2</power>
+								<chanceFactor>0.33</chanceFactor>
+								<cooldownTime>1.52</cooldownTime>
+								<armorPenetrationBlunt>0.500</armorPenetrationBlunt>
+								<linkedBodyPartsGroup>Handle</linkedBodyPartsGroup>
+							</li>
+							<li Class="CombatExtended.ToolCE">
+								<label>point</label>
+								<capacities>
+									<li>Stab</li>
+								</capacities>
+								<power>18</power>
+								<cooldownTime>1</cooldownTime>
+								<armorPenetrationBlunt>11.250</armorPenetrationBlunt>
+								<armorPenetrationSharp>90</armorPenetrationSharp>
+								<linkedBodyPartsGroup>Point</linkedBodyPartsGroup>
+							</li>
+							<li Class="CombatExtended.ToolCE">
+								<label>edge</label>
+								<capacities>
+									<li>Cut</li>
+								</capacities>
+								<power>46</power>
+								<cooldownTime>0.5</cooldownTime>
+								<armorPenetrationBlunt>45</armorPenetrationBlunt>
+								<armorPenetrationSharp>80</armorPenetrationSharp>
```

Comment:
I may consider reducing this further as compared to the stab, if only because for the vast, vast majority of enemies, this attack gets through every time, and there's no functional distinction between 90 and 80 Sharp AP. Moreover, at 46 damage, it'll be one-shot human targets. Perhaps the Royalty melee weapons may be a useful reference for something a _little_ less murderous?

Reply:
The description from the original mod indicates that this weapon could slice through human flesh "like nothing" or "like a hot knife through butter", but I found that I could halve the power and the sharp/blunt AP stats to still easily take down human pawns in 1~2 strikes, while still requiring 6~7 strikes for various Mechanoids.

Fixed in 512216f